src/BL/CurrecytManager.java

```diff
@@ -0,0 +1,51 @@
+package BL;
+
+import java.util.HashMap;
+
+public class CurrencyManager {
+    //Определяет статическую константу - значение валюты по умолчанию
+    public static final CurrencyTitle defaultCurrency = CurrencyTitle.EUR;
```

Comment:
this variable is a constant and it should be written in UPPER_SNAKE_CASE

Reply:
done